### ISSUE TITLE
License information needed

### ISSUE THREAD

**Issue by baranyaib90** [gryphius/syslog-stdout#1]
Hi,
Please provide the license of the code.
We would like to use it, but we need a valid license because of legal reasons.
Thank you in advance!
Br,
Balazs

Ps.: we would love some Freeware License :)

**Comment by gryphius** [comment]
I don't think there is anything in there that you wouldn't find in public documentation. SImilar code can be found in a mailing list post from 2003: http://twistedmatrix.com/pipermail/twisted-python/2003-July/004965.html . 
However, IANAL - I'm not sure if I'm allowed to put a license on such code snippets. If you can provide me with evidence that this is ok, I'll gladly put the project under a open license (MIT? Apache?)

**Comment by baranyaib90** [comment]
Hi,
I see.
In this case, i would suggest: https://choosealicense.com/licenses/unlicense/
Because I don't think, that the code author would reserve any rights.
And the code can be found in many part of the internet.
Sadly, we still need some license, to use this code.
(And we don't want to re-write this easy functionality.)
Do what you think it's best.
Br,
Balazs

Ps: I have searched, what is the "default" license on pypi and github.
I have found theese:
https://pypi.python.org/pypi?:action=user&otk=N5tQr7LvlblBnJ1R0OKZuO1xekPKCu9w
http://softwareengineering.stackexchange.com/questions/276144/using-unlicensed-code
